### DISCUSSION TITLE
Hi my brother, how about the .env VITE_API_URL?

### DISCUSSION THREAD

**Discussion by jhfnetboy** [jellydn/nft-app#601] [General]
VITE_CHAIN_ID=<YOUR CHAIN ID>
VITE_API_URL=YOUR_API_URL
I am confused about this API URL? how can I config it?
I run the repo locally and ADD Items with no response or error, the API URL is wrong(I give it a localhost address)

can you help me to run it properly?  thank you!

**Comment by jellydn** [comment]
Hi there. 
I'm quite busy right now so please watch my video about this repo https://www.youtube.com/watch?v=_veIZBaHkSk&list=PLOdXIcVPTyB-jyB4UmrwncHDK0mgl-Ex-&index=5

I will try to update README about it tonight or tomorrow. Thanks.

**Reply by jhfnetboy** [reply]
Ok, get it！ 
Thanks for your great jobs again~！
